test/test.helper.ts

```diff
@@ -6,6 +6,8 @@ import { Users } from 'src/entities/user.entity';
 import { HttpExceptionFilter } from 'src/http-exception-filter';
 import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { ConfigService } from '@nestjs/config';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const bcrypt = require('bcrypt');
```

Comment:
Is it possible to use `import { hash } from 'bcrypt' here too? 

Same in the `test/services/users.service.spec.ts` file

Reply:
Fix this comment! 🔧